guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,517 @@
 {
```

Comment:
<p>We don't want to publicize this yet as this release adds mostly the framework. Perhaps, in the next release the full functionality would come out and we can call this out then.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<ul><li>Removed note!</li></ul>